src/config.rs

```diff
@@ -70,6 +78,7 @@ mod tests {
     #[test]
     fn parse_nonempty() {
         let config = r#"
+docker_cli: docker
```

Comment:
The purpose of this test is to provide non-default values and ensure that they are parsed correctly (the `parse_empty` test above is for default values). So we should use something else here (like `podman`).